presto-product-tests/conf/docker/multinode/compose.sh

```diff
@@ -2,6 +2,8 @@
 
 source ${BASH_SOURCE%/*}/../common/compose-commons.sh
 
+export TEMPTO_CONFIG_YAML=$( canonical_path ${TEMPTO_CONFIG_YAML:-"./conf/tempto/tempto-configuration-for-docker-default.yaml"} )
```

Comment:
Also, please try some line breaks to make it read better (e.g. after `$(` ? probably some escaping is needed)